backend/app/models.py

```diff
@@ -4,3 +4,32 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 
 # create model classes here
+
+# Many-to-Many relationships requires helper table
+# According to the flask documentation
+contacts = db.Table(
+    "contacts",
+    db.Column("contact_id", db.Integer, db.ForeignKey("contact.id"), primary_key=True),
+    db.Column("project_id", db.Integer, db.ForeignKey("project.id"), primary_key=True),
+)
+# Project Model
+
```

Comment:
remove an extra line just for nicer formatting

Reply:
I think I've mentioned this but the extra line is added by black, I'll add this comment for future reference